### PR TITLE
create project structure

### LINKED ISSUE #1: Create project structure
TBD:
1. Create documentation about project structure based on best practices
2. (new) remove svelte kit and use svelte 

Add libs:
1. Add flowbite

### PR DESCRIPTION
**Pull request by Avareco**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,34 +1,38 @@
...
+    "axios": "1.7.7",
+    "lodash": "4.17.15",
+    "svelte-i18n": "4.0.0",
+    "svelte-spa-router": "4.0.1",
+    "yup": "0.32.11"
+  },
+  "devDependencies": {
+    "@sveltejs/adapter-auto": "^3.0.0",
+    "@sveltejs/vite-plugin-svelte": "^3.0.0",
+    "@types/eslint": "^9.6.0",
```
Comment (Aleshchenk0) [inline package.json]: I think now types are redundant due the typescript was deleted. Correct me if it uses for whatever.

File: src/App.svelte [hunk 2]
```diff
@@ -0,0 +1,10 @@
+<script>
+    import svelteLogo from './assets/svelte.svg';
+    import {Badge} from "flowbite-svelte";
```
Comment (Aleshchenk0) [inline src/App.svelte]: There are some moments when we have inconsistency in code. For example, single quotes and double quotes, some issues with tabulation so, run script `npm run format`, please. It'll fix it.

**Review by Aleshchenk0 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (Avareco) [reply package.json]: good catch, I will fix

**Comment by Avareco** [comment]
@Aleshchenk0 , @oleksandrstarov , I propose use pnpm instead of npm, as pnpm is faster, what do you think about it?

File: src/App.svelte (hunk 2 above)
Comment (Avareco) [reply src/App.svelte]: I'll add this script and corresponding rules

**Comment by oleksandrstarov** [comment]
@Avareco basically package management performance is not really a problem yet and probably wont be an issue at all, but if you really want to switch to pnpm I dont see any problems with this

**Comment by oleksandrstarov** [comment]
I see a good changes in direction we have discussed, nice

**Review by oleksandrstarov — APPROVED** [review]

**Review by Aleshchenk0 — APPROVED** [review]

**Pull request merged**